enzyme/Enzyme/Utils.h

```diff
@@ -562,30 +570,15 @@ static inline bool isCertainPrintMallocOrFree(llvm::Function *called) {
   if (called == nullptr)
     return false;
 
-  if (called->getName() == "printf" || called->getName() == "puts" ||
-      called->getName() == "fprintf" ||
-      called->getName().startswith("_ZN3std2io5stdio6_print") ||
-      called->getName().startswith("_ZN4core3fmt") ||
-      called->getName() == "vprintf" || called->getName() == "malloc" ||
+  if (isCertainPrintOrFree(called))
+    return true;
+
```

Comment:
Might as well have this also call isCertainMallocOrFree

Reply:
done